CHANGELOG.md

```diff
@@ -136,6 +136,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* (cli) [\#9593](https://github.com/cosmos/cosmos-sdk/pull/9593) Check if chain-id is blank before verifying signatures in multisign and error.
```

Comment:
could you move this to a new "improvements" subsection in the "Unreleased" section?